### PR TITLE
perf(linters): Greatly improve the `npm run lint` task speed

### PR DESCRIPTION
**Pull request by pixelastic**
The `npm run lint` task now takes about ~3s on my machine (was
taking XXX).

Previously:
```
$ time npm run lint
npm run lint  10,84s user 1,22s system 107% cpu 11,187 total
```
Now:
```
$ time npm run lint
npm run lint  3,73s user 0,28s system 116% cpu 3,445 total
```

Basically all I did was specify the `./**/*.js` or `./**/*.scss`
pattern on both linters and that makes it much faster.

For scss, I also cleaned up a bit the list of ignore patterns (as we
now use Jekyll 3, some files do not have the front-matter anymore and
so we do not need to exclude them).

### CODE REVIEW

File: scripts/lint-js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+eslint ./**/*.js --quiet
```
Comment (vvo) [inline scripts/lint-js]: Always wonder if the index.js is linted with this?

Comment (vvo) [inline scripts/lint-js]: Might not be a big deal, we could even run another command if needed

**Comment by pixelastic** [comment]
@vvo You were right that it wasn't checking at the root. I updated it

**Comment by vvo** [comment]
Thx :)

**Pull request merged**